### ISSUE TITLE
corner.jl:  ERROR: MethodError: no method matching dotc(::RGBA{Float64}, ::RGBA{Float64})

### ISSUE THREAD

**Issue by dbrowne** [JuliaImages/ImageFeatures.jl#65]
where is dotc defined?:  It is not the LinearAlgebra BLAS.dotc  
Thought using ColorVectorSpace.dotc might be a solution but to no avail.

function gradcovs(img::AbstractArray, border::AbstractString = "replicate"; weights::Function = meancovs, args...)
    (grad_x, grad_y) = imgradients(img, KernelFactors.sobel, border)

    cov_xx = dotc(grad_x, grad_x)
    cov_xy = dotc.(grad_x, grad_y)
    cov_yy = dotc.(grad_y, grad_y)

    weights(cov_xx, cov_xy, cov_yy, args...)
end
Any Suggestions?

**Comment by timholy** [comment]
Yes, it's ColorVectorSpace. If you can give a reproducible example of the problem you're having, it might help.
